bookmarks/frontend/components/SearchAutoComplete.svelte

```diff
@@ -209,9 +209,9 @@
     {/if}
     {#each suggestions.tags as suggestion}
       <li class="menu-item" class:selected={selectedIndex === suggestion.index}>
-        <a href="#" on:mousedown|preventDefault={() => completeSuggestion(suggestion)}>
+        <button on:mousedown|preventDefault={() => completeSuggestion(suggestion)}>
```

Comment:
Fixing this is unfortunately not as trivial:
![Bildschirmfoto 2024-03-16 um 15 37 41](https://github.com/sissbruecker/linkding/assets/357820/0fa9f0b9-542e-4a62-9b20-1d50ce809bd3)

The CSS library used here explicitly targets `.menu-item a`,  using any other element as child will not style it properly.

Reply:
Oops, my bad. I should've tested this more. I've reverted it now